app/ui.js

```diff
@@ -314,4 +295,99 @@ let productPopPrompt = () => {
       resolve(results);
     })
   });
+};
+
+module.exports.displayOrder = (total) => {
+  console.log(`
+  ${magenta('**  Your order total is,' total, 'Ready to purchase (Y/N)  **')}`);
```

Comment:
Hey, take a look at this line-- comma after is needs to be moved outside of the quotes.